tests/transforms/test_qmc_transform.py

```diff
@@ -262,7 +262,7 @@ def fn():
             qmc_circuit()
             qml.probs(estimation_wires)
 
-        tape = tape.expand()
+        tape = tape.expand(depth=2)
```

Comment:
Why is this needed?

Reply:
We need to add an additional layer of expansion to expand away `Adjoint` instances. Else we get `pennylane._device.DeviceError: Gate Adjoint(CNOT) not supported on device default.qubit` failure when running this test.